examples/huggingface/pytorch/text-generation/quantization/llm_quantization_recipes.md

```diff
@@ -0,0 +1,212 @@
+## Step-by-Step recipes for LLM quantization
+
+This document describes the step-by-step instructions to run large language models (LLMs) on 4th Gen Intel® Xeon® Scalable Processor (codenamed Sapphire Rapids) with [PyTorch](https://pytorch.org/) and [Intel® Extension for PyTorch](https://github.com/intel/intel-extension-for-pytorch).
+
+The scripts [run_generation.py](./run_generation.py) provide two quantization approaches respectively (SmoothQuant, Weight-Only Quantization) based on [Intel® Neural Compressor](https://github.com/intel/neural-compressor) and return last token prediction accuracy by `trainer`.
+
+# Prerequisite
+
+```bash
+# Installation
+git clone https://github.com/intel/intel-extension-for-transformers.git
+
+# install ITREX
+cd intel-extension-for-transformers
+git checkout a4aba8ddb07c9b744b6ac106502ec059e0c47960
+pip install -r requirements.txt
+pip install -v .
+
+# install requirements
+cd examples/huggingface/pytorch/text-generation/quantization
+pip install -r requirements.txt
+pip install git+https://github.com/intel/neural-compressor.git@v2.4.1rc
+pip install transformers==4.32.0
+pip install torch==2.1.1+cpu --index-url https://download.pytorch.org/whl/cpu
+pip install intel-extension-for-pytorch==2.1.100
+pip uninstall lm_eval -y
+pip install git+https://github.com/EleutherAI/lm-evaluation-harness.git@cc9778fbe4fa1a709be2abed9deb6180fd40e7e2
+```
+
+# Run Quantization and evaluate INT8 accuracy
```

Comment:
For each model, we need to cover INT4 WOQ as well.  Recipe has already available

Reply:
GPTQ relies on INC script, let's forget about this comments at this moment.  we can add INT4 WOQ after GPTQ quantization/evaluation is fully supported in ITREX